sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexAsyncClient.java

```diff
@@ -77,11 +75,11 @@
         this.serviceVersion = serviceVersion;
         this.httpPipeline = httpPipeline;
 
-        this.restClient = new SearchServiceRestClientBuilder()
+        this.restClient = new SearchServiceClientImplBuilder()
             .endpoint(endpoint)
-            .apiVersion(serviceVersion.getVersion())
+          //  .apiVersion(serviceVersion.getVersion())
```

Comment:
Any reason API version change was removed?

Reply:
Disregard, didn't realize new codegen prevent this from being set.

Reply:
This is what autorest v4 is intended to do. They currently agreed to only support one API version. Autorest team will follow up on this. Not API change, so just remove here